test/utils.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { ensureJSON } from "../lib/utils";
+import { JSONParseError } from "../lib/exceptions";
+import { equal, ok } from "assert";
+
+describe("utils", () => {
+  it("fails when it's not an object", () => {
```

Comment:
only `ensureJSON` is tested.

Reply:
only `ensureJSON` is uncovered so I only test the failed case.

Reply:
so maybe change to `ensureJSON fails ...` will be better?